frontend/packages/dev-console/src/components/custom-resource-list/CustomResourceList.tsx

```diff
@@ -42,80 +38,55 @@ const CustomResourceList: React.FC<CustomResourceListProps> = ({
 
       setListItems(newListItems || []);
       setFetched(true);
-
-      if (activeFilters) {
-        const filteredItems = rowFilterReducer(newListItems, activeFilters);
-        setFilteredListItems(filteredItems);
-      } else {
-        setFilteredListItems(newListItems);
-      }
     };
 
     fetchListItems();
 
     return () => {
       ignore = true;
     };
-  }, [dependentResource, fetchCustomResources, queryArg, rowFilters, rowFilterReducer]);
+  }, [dependentResource, fetchCustomResources]);
 
-  const applyRowFilter = React.useCallback(
-    (filter) => {
-      const filteredItems = rowFilterReducer(listItems, filter);
-      setFilteredListItems(filteredItems);
-    },
-    [listItems, rowFilterReducer],
-  );
+  const applyRowFilter = React.useCallback(() => {
```

Comment:
Probably better to rename this function since it handles both row filters and text filter change. 
```suggestion
  const applyFilters = React.useCallback(() => {
```